Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionAuthorsVC/ChatMessageReactionAuthorsFlowLayout.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// The default `ChatMessageReactionAuthorsVC` collection view flow layout.
+/// It manipulates the cell attributes so that each row is centred.
+class ChatMessageReactionAuthorsFlowLayout: UICollectionViewFlowLayout {
```

Comment:
shall we open it and add to `Components`?

Reply:
Actually, I made it open but didn't add it to components. It doesn't bring any value. If the customer wants to override the flow layout they can by subclassing the reaction authors VC. And if they change the authors vc to a table view for example, the flow layout is not even needed, so I think it is not necessary to add it to the components, and as far as I can see, we don't have the other flow layout on the components config